### PR TITLE
Update site.json (ko)

### PR DESCRIPTION
**Pull request by yous**
Added missing translations, and fixed wrong translations.

@nodejs/nodejs-ko Review please.

### CODE REVIEW

File: locale/ko/site.json [hunk 1]
```diff
@@ -105,13 +115,25 @@
       "link": "get-involved/code-and-learn",
       "text": "코딩 + 배우기"
     },
+    "collab-summit": {
+      "link": "get-involved/collab-summit",
+      "text": "Collab Summit"
```
Comment (ghost) [inline locale/ko/site.json]: Will this be also translated?
I mean the "text" of "collab-summit" :)

Comment (yous) [reply locale/ko/site.json]: I've thought up "Collab 서밋", please tell me what you think. @nodejs/nodejs-ko

Comment (yous) [reply locale/ko/site.json]: Maybe "협업 서밋" is appropriate.

**Comment by Maledong** [comment]
@nodejs/nodejs-ko：Please have a review, thanks!

File: locale/ko/site.json (hunk 1 above)
Comment (taggon) [reply locale/ko/site.json]: @yous Given the description in [the repository of the summit](https://github.com/nodejs/summit), "Collab" seems to stand for "Collaborators". How about "협업자 서밋"?

**Comment by taggon** [comment]
LGTM 👍

**Review by taggon — APPROVED** [review]

**Comment by ghost** [comment]
@yous, @taggon：Thank you both for your contributions! :)

File: locale/ko/site.json (hunk 1 above)
Comment (yous) [reply locale/ko/site.json]: I've only saw [Collab Summit](https://nodejs.org/en/get-involved/collab-summit/) page, and it says Collaboration Summit. Didn't know about the repository. The repo seems to be more accurate and the latest, so "협업자" would be good.

**Pull request merged**